packages/account/src/BiconomySmartAccount.ts

```diff
@@ -35,45 +34,32 @@ export class BiconomySmartAccount extends SmartAccount implements IBiconomySmart
     const {
       signer,
       rpcUrl,
-      epAddress,
+      entryPointAddress,
       factoryAddress,
-      bundlerUrl,
-      paymasterUrl,
-      nodeClientUrl,
-      dappApiKey,
-      userOpReceiptIntervals,
-      strictSponsorshipMode
+      bundler,
+      paymaster,
+      chainId,
+      nodeClientUrl
     } = biconomySmartAccountConfig
 
-    const _epAddress = epAddress ?? epAddresses.default
-    const _factoryAddress = factoryAddress ?? factoryAddresses.default
-    const _dappApiKey = dappApiKey ?? ''
-    const _strictSponsorshipMode = strictSponsorshipMode ?? false
-    const _paymasterUrl = paymasterUrl ?? ''
+    const _entryPointAddress = entryPointAddress ?? ENTRYPOINT_ADDRESSES.default
+    const _factoryAddress = factoryAddress ?? BICONOMY_FACTORY_ADDRESSES.default
     super({
-      bundlerUrl,
-      epAddress: _epAddress
+      bundler,
+      entryPointAddress: _entryPointAddress
     })
-    if (bundlerUrl)
-      this.bundler = new Bundler({
-        bundlerUrl,
-        epAddress: _epAddress,
-        apiKey: _dappApiKey,
-        userOpReceiptIntervals
-      })
-    this.provider = new JsonRpcProvider(rpcUrl)
-    this.entryPoint = EntryPoint_v100__factory.connect(_epAddress, this.provider)
+    const _rpcUrl = rpcUrl ?? RPC_PROVIDER_URLS[chainId]
```

Comment:
If developer pass a chainId that is not supported in SDK, we should throw an exception. 
`Chain Id ${chainId} is not supported. Supported chain Ids are ${array of supported chain ids}`

Reply:
resolved. Need to add supported chain list in documentation then will be able to provide link here.

Reply:
url is added in error for chain list